tokio-macros/src/entry.rs

```diff
@@ -174,6 +213,9 @@ fn parse_knobs(
                     "flavor" => {
                         config.set_flavor(namevalue.lit.clone(), namevalue.span())?;
                     }
+                    "start_paused" => {
+                        config.set_start_paused(namevalue.lit.clone(), namevalue.span())?;
+                    }
```

Comment:
The new attribute should be mentioned in the catch-all a few lines further down.

Reply:
Done